index.html

```diff
@@ -29,5 +29,122 @@ <h4>CONNECT WITH US</h4>
+                <li class="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </section>
+
+        <section class="work-1">
+            <div class="work-img"><img src="./assets/images/Snapshoot-Portfolio3.png" alt="Portfolio snapshoot"></div>
+            <div class="work-desc"><h1>Multi-Post
+                Stories</h1>
+            <ul class="frame">
+                <li class="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </section>
+        
+        <section class="description-1">
+            <h1>About<br>
+                Myself</h1>
+            <p>I’m a software developer! I can help you develop a product , feature or website Look through some of my work and experience! If you like what you see and have a project or content you to need create, don’t hestiate to contact me.</p>
+            <h4>CONNECT WITH US</h4>
+            <ul class="socials">
+                 <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                 <li><a href="#"><i class="fab fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fab fa-facebook"></i></a></li>
+                <li><a href="#"><i class="fab fa-github"></i></a></li>
+                <li><a href="#"><i class="fab fa-medium-m"></i></a></li>  
+            </ul>
+            <button class="button" type="button">Get my resume</button>
+            <ul class="first language">
+                <li ><h4>Languages</h4></li>
+                <li><i class="fas fa-angle-down"></i></li>
+            </ul>
+                <div class="list">
+                    <div class="lang">
+                        <img src="./assets/images/Ellipse-1.png" alt="js">
+                        <p>JavaScript</p>
+                    </div>
+                    <div class="lang">
+                        <img src="./assets/images/Ellipse-2.png" alt="html">
+                        <p>HTML</p>
+                    </div>
+                    <div class="lang">
+                        <img src="./assets/images/Ellipse-3.png" alt="css">
+                        <p>CSS</p>
+                    </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and li per icon; each icon has `a` link (a tag). Please use `ul` and `li `instead of `div`